packages/manager/modules/telecom-dashboard/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
This file isn't the correct one.

Reply:
Nope. Prefer using `present` instead of `$(date +%Y)`.

Furthermore GitHub provide a feature to auto-generate LICENSE file which is adding a header `BSD 3-Clause License`